crates/ruff/src/rules/pylint/rules/iteration_over_set.rs

```diff
@@ -38,23 +38,7 @@ impl Violation for IterationOverSet {
 
 /// PLC0208
 pub(crate) fn iteration_over_set(checker: &mut Checker, expr: &Expr) {
-    let is_set = match expr {
-        // Ex) `for i in {1, 2, 3}`
-        Expr::Set(_) => true,
-        // Ex)` for i in {n for n in range(1, 4)}`
-        Expr::SetComp(_) => true,
-        // Ex) `for i in set(1, 2, 3)`
-        Expr::Call(call) => {
-            if let Expr::Name(ExprName { id, .. }) = call.func.as_ref() {
-                id.as_str() == "set" && checker.semantic_model().is_builtin("set")
-            } else {
-                false
-            }
-        }
-        _ => false,
-    };
-
-    if is_set {
+    if let Expr::Set(_) = expr {
```

Comment:
Nit: can probably be `expr.is_set_expr()`.

Reply:
Fixed in [89c1dc3](https://github.com/charliermarsh/ruff/pull/4907/commits/89c1dc39f7cd47e9b8139bd3afa4574c01c60277)